### ISSUE TITLE
[Dropdown] Not fitting when placed inside scrolling modal

### ISSUE THREAD

**Issue by ko2in** [fomantic/Fomantic-UI#448]
# Bug Report
When the drop-down menu placed inside the scrolling modal, some parts of the drop-down menu covers by scrolling container and need to scroll from the scrolling container to see the rest part.

## Steps to reproduce
1. Add a modal and makes it to scroll internally
2. Add a medium or long list of drop-down menu in the modal. Place especially from middle to bottom half of scrolling area.
3. Click the drop-down menu to open

## Expected result
The drop-down menu should appear completely without covering the scrolling container.

![fomantic ui-dropdown inside scrolling modal](https://user-images.githubusercontent.com/930315/52059728-f6f4b600-2598-11e9-9af3-6c2028af4d6f.png)

## Actual result
The drop-down menu only appears certain area and has to scroll from the container to see the rest part.

![fomantic ui-dropdown inside scrolling modal-issue](https://user-images.githubusercontent.com/930315/52059643-c4e35400-2598-11e9-82a8-294b71aaf00e.png)

## Testcase
https://jsfiddle.net/n176sro4/

## Version
2.7.1

**Comment by hammy2899** [comment]
@ko2in This doesn't happen for me, what browser are you using?

**Comment by lubber-de** [comment]
That's the usual problem of `overflow-y`, because the dropdown-html is rendered inside the modal container. To fix this, we probably need to rewrite how dropdown works and make the dropdown list positioned absolute to the whole body. :thinking: I smell a huge change here ...

**Comment by ko2in** [comment]
Yes. I got this issue on Chrome (70.0.3538.110), and also on Firefox (63.0.3) on Ubuntu OS.

**Comment by dutrieux** [comment]
For me the actuel result is the right render. If we have expected result, the dropdown can overflow the dialog :
![overflow_modal](https://user-images.githubusercontent.com/1622751/52151970-14974d80-2675-11e9-85bf-efd3649448ef.png)
